compiler/next/include/chpl/queries/Context-detail.h

```diff
@@ -260,7 +260,7 @@ class QueryMap final : public QueryMapBase {
         // Keep the result
         ++iter;
       } else {
-        printf("Removing result %p %s\n", &result, queryName);
+        //printf("Removing result %p %s\n", &result, queryName);
```

Comment:
Cruft code? Should we just delete this line or do you like to keep these sorts of debugging printf statements around just so others know how to debug the code?